scripts/montagu-ci.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Montagu continuous integration VMs
+Wants=network-online.target
+After=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+WorkingDirectory=__PATH__
+User=__USER__
+ExecStart=__PATH__/scripts/start.sh
```

Comment:
I'm not sure if the issue here is in `start.sh` or in the service definition, but it would be nice if starting the service did not block (so that you start the service and it exists for a period in "booting" or some similar stage).  Otherwise every time we boot the machine it will take a while to start, especially if the machines have been removed

Reply:
It might be that `oneshot` is causing this waiting behaviour (from my reading of: https://www.freedesktop.org/software/systemd/man/systemd.service.html) - is there some other reason why oneshot is needed?

Reply:
I see it block when I run `start.sh` on the commandline, but not when booting the machine. This seems like helpful behaviour to me: It means that when it finishes in the terminal you know Vagrant should be fully up.

Reply:
:+1:  - I did not try a reboot but this seems like good behaviour